database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Models/Post.swift

```diff
@@ -14,32 +14,31 @@
 //  limitations under the License.
 //
 
-import UIKit
+import Foundation
 import Firebase
 
-struct Post {
+struct Post: Identifiable {
+  var id = UUID()
```

Comment:
Is this correct? In its current state, this implementation of Identifiable will treat two posts with the same author, uid, title, and body as different posts.

Reply:
I was initially using another class called PostModel but decided to just use UserViewModel instead. So I think the Identifiable property is no longer needed.

Reply:
Having an `id` variable is required, but instead of assigning it to a randomly-generated identifier, you should use the key from Firebase Database. This way, you can construct two instances of the same Post from a database query and they'll be treated as the same value by anything that observes the Identifiable protocol.